wizard_builder/models.py

```diff
@@ -3,21 +3,30 @@
 from polymorphic.models import PolymorphicModel
 
 from django import forms
-from django.core.exceptions import ObjectDoesNotExist
+from django.contrib.sites.models import Site
+from django.core.exceptions import ImproperlyConfigured, ObjectDoesNotExist
 from django.db import models
+from django.db.utils import OperationalError
 from django.utils.safestring import mark_safe
 
+from .managers import PageBaseManager
+
 
 def get_page_position():
     try:
         return PageBase.objects.latest('position').position + 1
-    except ObjectDoesNotExist:
+    except (ObjectDoesNotExist, OperationalError):
```

Comment:
i feel like i've tried this before, but is it worth just removing the default from `position` and retroactively editing the migration?